RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -290,10 +291,16 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 			costMinor = mp_cost($skill[Stream of Sauce]);
 			costMajor = mp_cost($skill[Stream of Sauce]);
 		}
-
-		if(!contains_text(combatState, "delaymortarshell") && canSurvive(2.0) && haveUsed($skill[Stuffed Mortar Shell]) && canUse($skill[Salsaball], false))
+		
+		#let mortar deal the killing blow so we get more MP from the exploding curse of weaksauce
+		int mortar_round = get_property("_auto_combatTracker_MortarRound").to_int();
+		if(mortar_round > -1 &&		//mortar was used this combat
+		mortar_round == round-1 &&	//mortar will hit this round
+		canSurvive(2.0) &&			//monster is not too scary to play games with
+		monster_hp() > 15 &&		//if monster has too few HP left we can accidentally kill it with salsaball and get only 2MP back
+		//TODO make sure mortar will actually kill it
```

Comment:
note I am working on this in combat2 branch by writing functions that actually calculate the damage we will be dealing with a skill when we cast it.